eslint/plugin/tests/src/rules/import-depth.js

```diff
@@ -17,18 +17,58 @@ const RuleTester = require('eslint').RuleTester;
 const ruleTester = new RuleTester();
 ruleTester.run('talend-import-depth', rule, {
 	valid: [
-		{ code: "import List from '@talend/react-components/lib/List';", parser: require.resolve('babel-eslint') },
-		{ code: "import SidePanel from '@talend/react-components/lib/SidePanel';", parser: require.resolve('babel-eslint') },
-		{ code: "import { SidePanel } from '@talend/react-components';", parser: require.resolve('babel-eslint') },
+		{
+			code: "import List from '@talend/react-components/lib/List';",
+			parser: require.resolve('babel-eslint'),
+		},
+		{
+			code: "import SidePanel from '@talend/react-components/lib/SidePanel';",
+			parser: require.resolve('babel-eslint'),
+		},
+		{
+			code: "import { SidePanel } from '@talend/react-components';",
+			parser: require.resolve('babel-eslint'),
+		},
+		{
+			code: "import cmf, { cmfConnect } from '@talend/react-cmf';",
+			parser: require.resolve('babel-eslint'),
+		},
+		{
+			code: "import stepper from '@talend/react-faceted-search';",
+			parser: require.resolve('babel-eslint'),
+		},
 	],
 
 	invalid: [
 		{
-            code: "import ListComposition from '@talend/react-components/lib/List/ListComposition';",
-            parser: require.resolve('babel-eslint'),
+			code: "import ListComposition from '@talend/react-components/lib/List/ListComposition';",
+			parser: require.resolve('babel-eslint'),
+			errors: [
+				{
+					message:
+						"'@talend/react-components/lib/List/ListComposition' import too deep. No more than /lib/List",
```

Comment:
Could be nice to have 
```suggestion
						"'@talend/react-components/lib/List/ListComposition' import too deep. No more than @talend/react-components/lib/List",
```
 instead of abs path

Reply:
done